scripts/index.js

```diff
@@ -57,8 +65,14 @@ const resolvedBuildPath = buildPath ? handleBuildPath(buildPath) : paths.appBuil
 // update the paths in config
 config.output.path = resolvedBuildPath;
 config.output.publicPath = publicPath || '';
-config.output.filename = `js/bundle.js`;
-config.output.chunkFilename = `js/[name].chunk.js`;
+
+// Grab output names from cli args, otherwise use some default naming.
+const fileNameToUse = outputFilename ? `${outputFilename}` : `js/bundle.js`;
+const chunkNameToUse = chunkFilename ? `${chunkFilename}` : `js/[name].chunk.js`;
```

Comment:
I think `outputFilename || 'js/bundle.js'` should do the trick. Same applies for chunkName